meta-lmp-base/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -1 +1,11 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+SRC_URI += "file://tmpfiles.conf"
+
+do_install:append () {
+    if ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)}; then
+        install -d ${D}${nonarch_libdir}/tmpfiles.d
+        install -m 0644 ${WORKDIR}/tmpfiles.conf ${D}${nonarch_libdir}/tmpfiles.d/${PN}.conf
+        rm -rf -v "${D}/${localstatedir}"
```

Comment:
As suggested by @quaresmajose at https://github.com/foundriesio/meta-lmp/pull/1092#pullrequestreview-1368623343, better to use rmdir instead.

Reply:
yeah, otherwise we can make the same issue removing content that may be necessary.